src/components/configuration/Configuration.ts

```diff
@@ -109,55 +109,43 @@ export class Configuration {
      */
     public async updatePathToRacer(): Promise<void> {
         const logger = this.logger.createChildLogger('updatePathToRacer: ');
+        this.racerPath = undefined;
 
-        this.pathToRacer = undefined;
-
-        const pathToRacerSpecifiedByUser: string | undefined = Configuration.getPathConfigParameter('racerPath');
-
-        const doesPathToRacerSpecifiedByUserExist: boolean = pathToRacerSpecifiedByUser
-            ? await FileSystem.doesFileOrDirectoryExists(pathToRacerSpecifiedByUser)
-            : false;
-
-        if (doesPathToRacerSpecifiedByUserExist) {
+        const userRacerPath: string | undefined = Configuration.getPathConfigParameter('racerPath');
+        if (userRacerPath != null && await FileSystem.which(userRacerPath) != null) {
```

Comment:
You are not explicit.
which returns `undefined`. If you want to be explicit, write `!== undefined`.